Kickstarter-iOS/AppDelegate.swift

```diff
@@ -234,6 +234,24 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         self?.rootTabBarController?.present(navController, animated: true)
       }
 
+    self.viewModel.outputs.verifyEmailWithURLRequest
+      .observeForUI()
+      .observeValues { [weak self] in self?.verifyEmail($0) }
+
+    self.viewModel.outputs.emailVerificationCompleted
+      .observeForUI()
+      .observeValues { [weak self] message, success in
+        // FIXME: We may want to revisit this shortcut to find the VC to present on.
+        guard
+          let firstVC = self?.rootTabBarController?.viewControllers?.first as? UINavigationController,
+          let topVC = firstVC.topViewController,
+          let vc = topVC as? MessageBannerViewControllerPresenting
+        else { return }
+
+        // FIXME: Consider emitting the success/error type from the output.
+        vc.messageBannerViewController?.showBanner(with: success ? .success : .error, message: message)
```

Comment:
We could emit `success` or `error` here too but the `Bool` seemed right to be at the time 🤷‍♂️

Reply:
Seems fine to me. Not sure what the trade off is for using a view model output?

Reply:
Nah this should be fine.